browser/src/control/Control.NavigatorPanel.ts

```diff
@@ -33,7 +33,9 @@ class NavigatorPanel extends SidebarBase {
 		super.onAdd(map);
 		this.map.on('navigator', this.onNavigator, this);
 		this.map.on('doclayerinit', this.onDocLayerInit, this);
-		this.map.on('focussearch', this.focusSearch, this);
+		if (this.map.isText()) {
```

Comment:
later we will add that tab to other apps. I is easy to reenable it.
What about checking if-null before use? :)
see `focusSearch` function